README.md

```diff
@@ -297,37 +324,25 @@ Make sure to have Node.js (At least v20) installed on your machine.
   ```
 
 - Run the test suites
-
+  
   ```sh-session
   npm run test
   npm run test-coverage # Run tests with coverage
   ```
 
-## Compatible specification types
+## License
 
-We currently support [OpenAPI](https://github.com/OAI/OpenAPI-Specification) from 2.0 (called Swagger) to 3.1 and [AsyncAPI 2.x](https://www.asyncapi.com/docs/reference/specification/latest) specification file types. Both YAML and JSON file formats are accepted file inputs to the CLI.
+The Bump CLI project is released under the [MIT License](http://opensource.org/licenses/MIT).
 
 ## Contributing
 
 Bug reports and pull requests are welcome on GitHub at <https://github.com/bump-sh/cli>. This project is intended to be a safe, welcoming space for collaboration, and contributors are expected to adhere to the [Contributor Covenant](http://contributor-covenant.org) code of conduct.
 
-## Thanks
-
-- [Lorna Mitchel](https://github.com/lornajane/) for [openapi-overlay-js](https://github.com/lornajane/openapi-overlays-js)
-
-## License
-
-The Bump CLI project is released under the [MIT License](http://opensource.org/licenses/MIT).
-
 ## Code of Conduct
 
 Everyone interacting in the Bump-CLI project codebases, issue trackers, chat rooms, and mailing lists is expected to follow the [code of conduct](https://github.com/bump-sh/.github/blob/main/CODE_OF_CONDUCT.md).
 
-## Versioning
-
-This npm package starts at v2.0.0 for two main reasons:
-
-- Our [first version](https://github.com/bump-sh/bump-cli) of the Bump CLI was written in Ruby, starting at v2.0.0, which makes it clear we are working on our second version of the Bump CLI
+## Thanks
 
-- The `bump-cli` package used to be [owned by Rico](https://github.com/rstacruz) which already published v1.x packages. If you are looking for the old npm package please head to [`@rstacruz/bump-cli` package](https://www.npmjs.com/package/@rstacruz/bump-cli). _A big thanks to Rico for transferring the ownership of the `bump-cli` package name!_
```

Comment:
This context was probably helpful at the time but its four years ago now, and is something that could be tidied away. I've put a quick line into Thanks to keep the sentiment but with a lot less space used up.

Reply:
:+1: looks good, thanks :bow: